nvflare/apis/impl/job_def_manager.py

```diff
@@ -237,6 +236,28 @@ def get_content(self, meta: dict, fl_ctx: FLContext) -> Optional[bytes]:
         except StorageException:
             return None
 
+    def _save_log(self, jid: str, log: str, client_name: str, log_type: str, fl_ctx: FLContext):
+        """Save the provided log content for the specified job, client name and log type."""
+        store = self._get_job_store(fl_ctx)
+        log_object_type = f"{log_type}_{client_name}"
+        store.update_object(self.job_uri(jid), log.encode(), log_object_type)
+
+    def _get_log(self, jid: str, client_name: str, log_type: str, fl_ctx: FLContext) -> Optional[str]:
+        """Get log content for the specified job, client name and log type."""
+        store = self._get_job_store(fl_ctx)
+        log_object_type = f"{log_type}_{client_name}"
+        try:
+            log_data = store.get_data(self.job_uri(jid), log_object_type)
+            return log_data.decode()
+        except StorageException:
+            return None
+
+    def set_log(self, jid: str, log: str, client_name: str, log_type: str, fl_ctx: FLContext):
+        self._save_log(jid, log, client_name, log_type, fl_ctx)
```

Comment:
suggest "set_client_log", "_save_client_log", "_get_client_log" if "client_name" is a required argument